runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -274,4 +276,21 @@ public static int PACK(int groupId, int childId)
 		return groupId << 16 | childId;
 	}
 
+	private static final HashMap<Integer, WidgetInfo> idMap = new HashMap<>();
+
+	public static WidgetInfo getWidgetInfo(int packedId)
+	{
+		if(idMap.size() == 0)
```

Comment:
It may be better to just place this in the widget tree stuff since I think there are no legitimate uses otherwise?